montblanc/factory.py

```diff
@@ -150,7 +150,8 @@ def create_biro_solver_from_ms(slvr_class_type, **kwargs):
     with MeasurementSetLoader(kwargs.get('msfile')) as loader:
         ntime,na,nchan = loader.get_dims()
         slvr = slvr_class_type(na=na,ntime=ntime,nchan=nchan,**kwargs)
-        loader.load(slvr)
+        kwargs['uvw_order']=kwargs.get('uvw_order')
```

Comment:
> If I don't do this, the load function is not able to find this keyword. I mean, this keyword is read by the BaseSolver but I need to pass it to the loader.

@marziarivi. OK the full diff here is:
```python
kwargs['uvw_order']=kwargs.get('uvw_order')
loader.load(slvr, **kwargs)
```
I agree that you need the second line to pass the kwargs dictionary through to the load function. But I still don't think the first line is necessary.